spec/models/miq_queue_spec.rb

```diff
@@ -54,29 +54,26 @@
     end
 
     it "passes args" do
-      # not a valid method, just making sure everything is passed
-      expect(MiqServer).to receive(:my_zone).with("1", "2").and_return("MY ZONE")
```

Comment:
In this case, it's only the args that are not valid, so I'd prefer not to change the class...

Reply:
I couldn't find a method that took at least two arguments in MiqServer, so that's why I changed it.

Reply:
Ok, updated.